gradio/themes/base.py

```diff
@@ -452,7 +452,7 @@ def set(
         button_small_text_size=None,
         button_small_text_weight=None,
         button_transition=None,
-    ):
+    ) -> Base:
```

Comment:
Added this typehint for better IDE completion on `push_to_hub`, `dump` methods after `set`

Reply:
Thanks @freddyaboulton!